packages/logging_cloudwatch/amplify_logging_cloudwatch/example/lib/homepage.dart

```diff
@@ -10,40 +10,51 @@ class MyHomePage extends StatefulWidget {
   State<MyHomePage> createState() => _MyHomePageState();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  final _plugin = AmplifyLogger().getPlugin<AmplifyCloudWatchLoggerPlugin>()!;
-
-  final _authLogger = AmplifyLogger.category(Category.auth);
-  LogLevel? _authLogLevel;
-  final TextEditingController _authLogMsgController = TextEditingController();
-
-  final _analyticsLogger = AmplifyLogger.category(Category.analytics);
-  LogLevel? _analyticsLogLevel;
-  final TextEditingController _analyticsLogMsgController =
-      TextEditingController();
+class LoggerInfo {
+  LoggerInfo(
+    this.category,
+    this.logger,
+    this.logMsgController, {
+    this.logLevel = LogLevel.verbose,
```

Comment:
nit: better to not set a default value so the log level would be null if not set by user.

Reply:
Is that desired behavior though?  Since if it's null the logic prevents any log from being sent while wiping the user's log message that they wrote?

Reply:
good observation. they way I see it is if the log level is not set then user would see it and set the value. given the app does not show error if user does not set the log level but put a log message and wipes out the log message it is a good idea to set the log level to vebose.